### PR TITLE
Adds distant explosion sounds

### PR DESCRIPTION
**Pull request by NoooneyDude**
- Adds distant explosion and creak sounds from /tg/.
    - These sounds are global and only trigger with a large enough explosion, such as shuttle collisions, welding fuel tanks, reactor meltdowns and sometimes the syndicate minibomb.
    - The local explosion sound is no longer global. This should significantly reduce ear strain when there is a wizard lurking about or a crazed botanist. A partial fix for #4578.
- Fixes deconstructing multiple disposal pipes at a time resulting in only the last pipe to be deconstructed.
- Fixes the disposal bent pipes spawning the wrong object orientation when deconstructing the tile.
- Updates mobs in `CentCom` scene to new prefab variants where possible.

CL:Fixed deconstructing disposal pipes, added distant explosion sounds

### CODE REVIEW

**Comment by Bod9001** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- UnityProject/Assets/Scripts/Items/Weapons/ExplosionUtils.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6394885&bid=20738283)

File: UnityProject/Assets/Scripts/Items/Weapons/ExplosionUtils.cs [hunk 1]
```diff
@@ -20,7 +24,18 @@ public static class ExplosionUtils
 		public static void PlaySoundAndShake(Vector3Int worldPosition, byte shakeIntensity, int shakeDistance)
 		{
 			string sndName = EXPLOSION_SOUNDS[Random.Range(0, EXPLOSION_SOUNDS.Length)];
-			SoundManager.PlayNetworkedAtPos(sndName, worldPosition, -1f, true, true, shakeIntensity, (int)shakeDistance);
+			SoundManager.PlayNetworkedAtPos(sndName, worldPosition, -1f, true, true, shakeIntensity, (int)shakeDistance, false);
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Items/Weapons/ExplosionUtils.cs:27]: Codacy found an issue: [Remove this unnecessary cast to 'int'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6394885)

**Comment by Bod9001** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- UnityProject/Assets/Scripts/Items/Weapons/ExplosionUtils.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6394885&bid=20738283)

**Comment by corp-0** [comment]
oh you beautiful man, this was something I was missing so much

**Pull request merged**